app/src/main/cpp/Skybox.cpp

```diff
@@ -254,6 +254,11 @@ FileDoesNotExist (const std::string& aName) {
 std::string
 Skybox::ValidateCustomSkyboxAndFindFileExtension(const std::string& aBasePath) {
   for (const std::string& ext: sFileExt) {
+#if defined(PICOXR)
+     // FIXME: Pico does not support compressed textures yet.
+     if (ext == ".ktx")
+       continue;
+#endif
```

Comment:
This is not going to work because that method searches files in a different path. It's used for custom environments, i.e, environments that are added by the user directly to the wolvic data directory.

Reply:
We already have docs for custom environments [here](https://github.com/Igalia/wolvic/wiki/Environments)